backend/src/nodes/image_nodes.py

```diff
@@ -135,12 +135,12 @@ def __init__(self):
         self.side_effects = True
 
     def run(
-        self,
-        img: np.ndarray,
-        base_directory: str,
-        relative_path: Union[str, None],
-        filename: str,
-        extension: str,
+            self,
+            img: np.ndarray,
+            base_directory: str,
+            relative_path: Union[str, None],
+            filename: str,
+            extension: str,
```

Comment:
Please revert these formatting changes.

Also, we use black for formatting, so please run it.